sdk/src/types/block/output/feature/mod.rs

```diff
@@ -431,10 +431,10 @@ pub mod dto {
                 FeatureDto::Sender(v) => Self::Sender(SenderFeature::new((&v.address).try_into()?)),
                 FeatureDto::Issuer(v) => Self::Issuer(IssuerFeature::new((&v.address).try_into()?)),
                 FeatureDto::Metadata(v) => Self::Metadata(MetadataFeature::new(
-                    prefix_hex::decode(&v.data).map_err(|_e| Error::InvalidField("MetadataFeature"))?,
+                    prefix_hex::decode::<Vec<u8>>(&v.data).map_err(|_e| Error::InvalidField("MetadataFeature"))?,
```

Comment:
why are we doing `_e`, and not just `_` if we don't care about the error?

Reply:
No specific reason

Reply:
Feel free to change later if you want, out of scope here